crates/fluvio-test/src/tests/producer.rs

```diff
@@ -100,30 +104,30 @@ async fn producer_work(
         //  Create record
         //  calculate the min latency to meet throughput requirements
         //
-        //  start time
-        //  (dynamic wait based on previous iteration)
+        //  start time=now
         //  send
-        //  end time
+        //  end time=now
         //
         //  calculate actual latency, throughput
         // }
 
-        let record = TestRecordBuilder::new()
-            .with_tag(format!("{}:{}", producer_id, record_n))
-            .with_random_data(test_case.option.record_size)
-            .build();
-        let record_json = serde_json::to_string(&record)
-            .expect("Convert record to json string failed")
-            .as_bytes()
-            .to_vec();
-        let record_size = record_json.len() as u64;
+        // This bottlenecks the system, not good for throughput test
```

Comment:
if the code is not needed so, let's just remove it instead of commenting

Reply:
:+1:. It looked like the file was still underdevelopment and I wasn't sure if there was a vision for it in the broader context.